src/main/resources/com/atlassian/bitbucket/jenkins/internal/jenkins/oauth/servlet/AuthorizeConfirmationConfig/index.jelly

```diff
@@ -2,31 +2,62 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
     <l:layout permission="${app.READ}">
-        <l:main-panel>
-            <h2>Welcome to ${it.instanceName}</h2>
-            <p>
-                The application <strong>${it.consumerName}</strong> would like to have <strong>${it.accessRequest}</strong>
-                access to your data on <strong>${it.urlName}</strong>. The application will be allowed to use your
-                credentials and authenticate as you (<strong>${it.authenticatedUsername}</strong>) in the future.
-            </p>
+        <style type="text/css">
+            .dialog-container {
+                text-align: center;
+                max-width: 678px;
```

Comment:
These numbers (and the colours) appear to be random. I'm sure they're not, so can we use css variables to name them?